hardhat/scripts/start-ganache.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# read .env, filtering out comments
+export $(cat ../.env | sed 's/^#.*$//' | xargs)
+
+exec ./node_modules/.bin/ganache \
+  -p 8555 \
```

Comment:
TODO: parameterize this... the idea is to avoid using the default port, so a developer can be using one testchain instance for persistent local data, & running another one on a different port just for tests